.drone.env

```
@@ -1,3 +1,3 @@
 # The version of OCIS to use in pipelines that test against OCIS
-OCIS_COMMITID=a5eb9611c07a9131128019228c77ab2f9daeeb4a
+OCIS_COMMITID=1f53ec5253c00d7e33092a68bdc37c5a859b4567
```

@grgprarup Could you bump to this latest commit `b44b33e9f821fec2f08e34d1516660759ee6d44b`. Meanwhile, we can check if using `nodejs:16` image works or not
```suggestion
OCIS_COMMITID=b44b33e9f821fec2f08e34d1516660759ee6d44b
```